tutor/src/screens/assignment-edit/details-body.js

```diff
@@ -168,6 +168,26 @@ const ExternalUrlField = observer(({ ux }) => {
   );
 });
 
+const ClonedAssignmentNotes = styled(({ ux, className }) => {
+  if (!ux.showPreWRMCloneHelp) { return null; }
+  return (
+    <div className={className}>
+      <div><b>Note</b></div>
+      <ul>
+        <li>OpenStax Tutor Beta allows you to assign points to questions.</li>
+        <li>You can assign Written Response Questions (manually-graded) to students.</li>
+        <li>By default, MCQs are worth 1 point, and WRQs are worth 2 points.</li>
+      </ul>
+    </div>
+  );
+})`
+  color: ${colors.neutral.dark};
+  ul {
+    margin: 0;
+    padding-left: 3rem;
+  }
+`;
+
```

Comment:
@camliu89 and @jivey - what do you think of this pattern?  It lets us avoid creating a new styled component